managed/utils/envvars/parser.go

```diff
@@ -179,6 +179,9 @@ func ParseEnvVars(envs []string) (envSettings *models.ChangeSettingsParams, errs
 			// This variable is not part of the settings and is parsed separately.
 			continue
 
+		case "DISABLE_INTERNAL_DB":
```

Comment:
That seems to be ambiguous given that we have three "internal" databases. How about "PMM_DISABLE_BUILTIN_POSTGRES"?

Reply:
env renamed

Reply:
Looks like not all of them. Can we create dedicated type for envs and use constants instead of strings?